packages/p2p/src/components/verification/verification.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
+import { observer } from 'mobx-react-lite';
 import { Icon, Checklist, Text } from '@deriv/components';
 import { isMobile, routes } from '@deriv/shared';
-import { observer } from 'mobx-react-lite';
-import { useStores } from 'Stores';
 import Dp2pBlocked from 'Components/dp2p-blocked';
 import { Localize } from 'Components/i18next';
-import './verification.scss';
+import { useStores } from 'Stores/index';
 
-const VerificationWrapper = ({ should_wrap, children }) => {
+const VerificationWrapper = ({ should_wrap, children }: React.PropsWithChildren<{ should_wrap: boolean }>) => {
```

Comment:
suggestion(non-blocking): should we use a type for this instead? Maybe `TVerificationWrapperProps` just to keep it consistent